src/pe_reports/report_generator.py

```diff
@@ -22,21 +43,49 @@ def load_template():
 
 def export_set(prs):
     """Export PowerPoint report set to output directory."""
-    pptx_out = "Customer_ID_Posture_Exposure.pptx"
-    prs.save(os.path.join(SAVE_PATH, pptx_out))
+    try:
+        pptx_out = "Customer_ID_Posture_Exposure.pptx"
+        prs.save(os.path.join("/output", pptx_out))
+    except OSError:
+        print("No output available.")
     return
 
 
+def generate_reports(data, data_dir, out_dir):
+    """Gather assets to produce reports."""
+    # TODO: build code to connect customer db, encrypt and embed pdf reports.
+    # Issue #7: https://github.com/cisagov/pe-reports/issues/7
+
+
 def main():
+    """Set up logging and call the generate_reports function."""
+    args: Dict[str, str] = docopt.docopt(__doc__, version=__version__)
+
+    # Set up logging
+    logging.basicConfig(
+        format="%(asctime)-15s %(levelname)s %(message)s", level=logging.INFO
+    )
```

Comment:
Since you are now taking the logging level as a command line option you need to use it in your logging setup.

```suggestion
    # Set up logging
    logging.basicConfig(
        format="%(asctime)-15s %(levelname)s %(message)s",
        level=args["--log-level"].upper(),
    )
```

Reply:
Resolved in 253b2dd

Reply:
Resolved in ecea2ce